csv/src/test/java/com/fasterxml/jackson/dataformat/csv/ser/TestGenerator.java

```diff
@@ -119,6 +120,21 @@ public void testExplicitWithFloat() throws Exception
         assertEquals("abc,1.89\n", result);
     }
 
+    public void testExplicitWithFastFloat() throws Exception
+    {
+        CsvSchema schema = CsvSchema.builder()
+                .addColumn("id")
+                .addColumn("amount")
+                .build();
+
+        float amount = 1.89f;
+        //this value loses precision when converted
```

Comment:
Not sure I fully understand the comment here?

Reply:
this new test is a copy/paste of the test above it and that pre-existing test has this comment too - I will remove the comment from both tests

Reply:
SG. Let me know when all done, I'll merge this.